ang/civicase/CaseOverview.js

```diff
@@ -81,9 +83,32 @@
     };
 
     /**
-     * Toggle show breakdown dropdown
+     * Checks if all statuses are disabled
+     *
+     * @return {Boolean}
+     */
+    $scope.isAllStatusesDisbabled = function () {
+      return _.filter($scope.caseStatuses, function (status) {
+        return !status.disabled;
+      }).length === 0;
+    };
+
+    /**
+     * Toggle status view
+     *
+     * @param {event} event object
+     * @param {Number} index of the case status
      */
-    $scope.showHideBreakdown = function () {
+    $scope.toggleStatusView = function ($event, index) {
```

Comment:
We are toggling the visibility right? then what do you think about `toggleStatusVisibility`?